src/lib.rs

```diff
@@ -161,25 +163,19 @@ pub fn run_wasm_with_css(css: &str) {
 
     // build wasm example via cargo
     let cargo = env::var("CARGO").unwrap_or_else(|_| "cargo".to_string());
-    let project_root = Path::new(&std::env::var("CARGO_MANIFEST_DIR").unwrap())
-        .ancestors()
-        .nth(1)
-        .unwrap()
-        .to_path_buf();
-    let mut cargo_args = vec![
-        "build",
-        "--target",
-        "wasm32-unknown-unknown",
-        // It is common to setup a faster linker such as mold or lld to run for just your native target.
-        // It cant be set for wasm as wasm doesnt support building with these linkers.
-        // This results in a separate rustflags value for native and wasm builds.
-        // Currently rust triggers a full rebuild every time the rustflags value changes.
-        //
-        // Therefore we have this hack where we use a different target dir for wasm builds to avoid constantly triggering full rebuilds.
-        // When this issue is resolved we might be able to remove this hack: https://github.com/rust-lang/cargo/issues/8716
-        "--target-dir",
-        "target/wasm-examples-target",
-    ];
+
+    let project_root = Path::new(&std::env::var("CARGO_MANIFEST_DIR").unwrap()).to_path_buf();
```

Comment:
project_root is used as the working directory for cargo build. Changing the working directory like this breaks cargo-run-wasm as it is used in other projects such as wgpu. I would recommend verifying that wgpu or winits use of cargo-run-wasm continues to function before pushing again.

Reply:
This surprises me. My mental model of cargo's building model within workspaces is that it doesn't matter where in the workspace the command is run, it always has the same effect.

In my mind, choosing an arbitrary directory as working directory is strange. The only information we have about which workspace we are in is `CARGO_MANIFEST_DIR`, so it seemed that we should use this.

I'll see where my mental model breaks down by trying this in winit and wgpu.